contributing/development.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Contributing to Libera Chat
+---
+
+- Help out with issues and PRs in [Libera-Chat/libera-chat.github.io](https://github.com/Libera-Chat/libera-chat.github.io)
+- Make artwork and make sure to choose an appropriate license (at least CC-BY-SA)
+- Be helpful and nice to others and maybe even assist people who need help
+- Contribute to the technologies powering Libera Chat, which include:
+  - [Solanum](https://github.com/solanum-ircd/solanum) - the IRCd powering the network
```

Comment:
Wonder if it also makes sense to link to solanum.chat and atheme.dev

Reply:
Both addresses are already in the repositories which were linked. I would either leave the addresses as they are right now or replace them with the actual websites of the project. Since this is about contributing, I would maybe use the repository link.

Reply:
Sgtm